cvat/apps/dataset_manager/tests/test_formats.py

```diff
@@ -484,6 +576,498 @@ def test_frames_outside_are_not_generated(self):
+                self._download_file(url, data, self.admin, file_zip_name)
+                self._check_downloaded_file(file_zip_name)
+
+                # remove annotations
+                response = self._delete_request(url, self.admin)
+                self.assertEqual(response.status_code, status.HTTP_204_NO_CONTENT)
+
+                # upload annotations
+                if dump_format_name in ["CVAT for images 1.1", "CVAT for video 1.1"]:
+                    upload_format_name = "CVAT 1.1"
+                else:
+                    upload_format_name = dump_format_name
+
+                url = f"/api/v1/tasks/{task_id}/annotations?format={upload_format_name}"
+                with open(file_zip_name, 'rb') as binary_file:
+                    response = self._upload_file(url, binary_file, self.admin)
+                    self.assertEqual(response.status_code, status.HTTP_202_ACCEPTED)
+
+                # equals annotations
+                task_ann = TaskAnnotation(task_id)
+                task_ann.init_from_db()
+                task_data = TaskData(task_ann.ir_data, Task.objects.get(pk=task_id))
+                extractor = CvatTaskDataExtractor(task_data)
+                data_from_task_after_upload = datumaro.components.project.Dataset.from_extractors(extractor)
+                compare_datasets(self, data_from_task_before_upload, data_from_task_after_upload)
+
+    def test_api_v1_tasks_annotations_update_wrong_label(self):
+        test_name = self._testMethodName
+        dump_format_name = "CVAT for images 1.1"
+        upload_format_name = "CVAT 1.1"
+
+        # create task with annotations
+        images = self._generate_task_images(3)
+        task = self._create_task(tasks["main"], images)
+        self._create_annotations(task, dump_format_name, "dafault")
+        task_id = task["id"]
+        task_ann = TaskAnnotation(task_id)
+        task_ann.init_from_db()
+
+        # dump annotations
+        url = f"/api/v1/tasks/{task_id}/annotations"
+        data = {
+            "format": dump_format_name,
+            "action": "download",
+        }
+        file_zip_name_before_change = osp.join(osp.dirname(__file__), 'assets', f'{test_name}_{dump_format_name}.zip')
+        file_zip_name_after_change = osp.join(osp.dirname(__file__), 'assets', f'{test_name}_{dump_format_name}_wrong_label.zip')
+
+        # download zip file
+        self._download_file(url, data, self.admin, file_zip_name_before_change)
+        self._check_downloaded_file(file_zip_name_before_change)
+
+        # remove annotations
+        response = self._delete_request(url, self.admin)
+        self.assertEqual(response.status_code, status.HTTP_204_NO_CONTENT)
+
+        # extract zip
+        folder_name = osp.join(osp.dirname(__file__), 'assets', f'{test_name}_{dump_format_name}')
+        with zipfile.ZipFile(file_zip_name_before_change, 'r') as zip_ref:
```

Comment:
Consider using `TestDir()` like here: https://github.com/openvinotoolkit/datumaro/blob/develop/tests/test_datumaro_format.py#L84

Otherwise, all the occasional trash from tests is going to be stored in the launch directory, can be intermixed from different tests, with the actual code etc.

Reply:
Done. Added it actions.